examples/lighting-app/efr32/src/Rpc.cpp

```diff
@@ -33,12 +36,58 @@
 namespace chip {
 namespace rpc {
 
-class LightingService final : public generated::LightingService<LightingService>
```

Comment:
Why did `final` get removed?

Reply:
Is just not part of the Pigweed RPC stubs, added them back.